modules/openapi-generator/src/main/resources/tiny-cpp-client/service/api-header.mustache

```diff
@@ -0,0 +1,74 @@
+#ifndef _{{classname}}_H_
+#define _{{classname}}_H_
+
+{{{defaultInclude}}}
+#include "Response.h"
+#include "Arduino.h"
+#include "AbstractService.h"
+#include "Helpers.h"
+#include <list>
+
+{{#imports}}
+{{{import}}}
+{{/imports}}
+
+namespace Tiny {
+
+{{#operations}}
+/**
+ *  Class {{basename}}
+ * Generated with openapi::tiny-cpp-client
+ */
+
+class {{classname}} : public AbstractService {
+public:
+    {{! Constructor }}
+    {{classname}}();
```

Comment:
Constructor and destructor are always empty - they could be set to default and removed from api-body.mustache.

```
{{classname}}() = default;
virtual ~{{classname}}() = default;
```